install_config/router/f5_router.adoc

```diff
@@ -35,11 +34,8 @@ HAProxy router].
 
 [IMPORTANT]
 ====
-Support relationships between F5 and Red Hat provide a full scope of support for
-F5 integration. F5 provides support for the *F5 BIG-IP®* product. Both F5 and
-Red Hat jointly support the integration with Red Hat OpenShift. While Red Hat
-helps with bug fixes and feature enhancements, all get communicated to F5
-Networks where they are managed as part of their development cycles.
+Support relationships between F5 and Red Hat provide a full scope of support for both models of 
+ F5 integration (F5 router plug-in and the F5 BIG-IP Controller for OpenShift).
```

Comment:
Remove the space before F5.
s/integration (F5 router plug-in and the F5 BIG-IP Controller for OpenShift)./integration, F5 router plug-in and the F5 BIG-IP Controller for OpenShift.

Reply:
I'm fine with these changes